From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Pascalpex <pascalpex@gmail.com>
Date: Mon, 21 Jul 2025 11:43:33 +0200
Subject: [PATCH] Rebrand


diff --git a/src/main/java/org/bukkit/craftbukkit/util/Versioning.java b/src/main/java/org/bukkit/craftbukkit/util/Versioning.java
index c3d45e60e262b8e34d76f0425f6bef79f034e8ab..79fe203e2ca9b3e4f515eb65d9b240a6022c9fdb 100644
--- a/src/main/java/org/bukkit/craftbukkit/util/Versioning.java
+++ b/src/main/java/org/bukkit/craftbukkit/util/Versioning.java
@@ -11,7 +11,7 @@ public final class Versioning {
     public static String getBukkitVersion() {
         String result = "Unknown-Version";
 
-        InputStream stream = Bukkit.class.getClassLoader().getResourceAsStream("META-INF/maven/cn.dreeam.leaf/leaf-api/pom.properties"); // Gale - branding changes // Leaf - Rebrand
+        InputStream stream = Bukkit.class.getClassLoader().getResourceAsStream("META-INF/maven/cn.dreeam.leaf/nycticorax-api/pom.properties"); // Gale - branding changes // Leaf - Rebrand // Nycticorax - Rebrand
         Properties properties = new Properties();
 
         if (stream != null) {
